pkg/cmd/delete/delete.go

```diff
@@ -23,7 +21,6 @@ func NewCmd(f *cmdutil.Factory) *cobra.Command {
 	}
 
 	cmd.AddCommand(edgeApplication.NewCmd(f))
-	cmd.AddCommand(ruleEngine.NewCmd(f))
```

Comment:
add this back :) 
As well as delete/rules_engine.go and delete/rules_engine_test.go